Adyen.EcommLibrary.Test/BaseTest.cs

```diff
@@ -142,6 +142,28 @@ protected Model.Checkout.PaymentMethodsRequest CreatePaymentMethodRequest(string
         {
             return new Model.Checkout.PaymentMethodsRequest(MerchantAccount: merchantAccount);
         }
+
+        /// <summary>
+        /// Checkout paymentsessionRequest
+        /// </summary>
+        /// <returns></returns>
+        protected Model.Checkout.PaymentSessionRequest CreatePaymentSessionRequest()
+        {
+            return new Model.Checkout.PaymentSessionRequest(MerchantAccount: "MerchantAccount", Reference: "MerchantReference",
+                 Amount: new Model.Checkout.Amount("EUR", 1200), ReturnUrl: @"https://your-company.com/...", CountryCode: "NL",
+                 Channel: Model.Checkout.PaymentSessionRequest.ChannelEnum.Web, SdkVersion: "1.3.0");

```

Comment:
SdkVersion should not be needed for web right ?

Reply:
It is required field for web as well

Reply:
ok, clear.